### ISSUE TITLE
TypeError: Cannot read property 'aggregate' of undefined

### ISSUE THREAD

**Issue by sudhanshugaur4** [vkarpov15/mongoose-explain#3]
```
/project/node_modules/mongoose-explain/index.js:36
data:    app.js:1037 -   if (pluginOptions.aggregate !== false) {
data:    app.js:1037 -                    ^
data:    app.js:1037 - TypeError: Cannot read property 'aggregate' of undefined
data:    app.js:1037 -     at module.exports (/project/node_modules/mongoose-explain/index.js:36:20)
data:    app.js:1037 -     at Schema.plugin (/project/node_modules/mongoose/lib/schema.js:1226:3)
data:    app.js:1037 -     at Object.<anonymous> (/project/models/trans_model.js:43:25)
data:    app.js:1037 -     at Module._compile (module.js:577:32)
data:    app.js:1037 -     at Object.Module._extensions..js (module.js:586:10)
data:    app.js:1037 -     at Module.load (module.js:494:32)
data:    app.js:1037 -     at tryModuleLoad (module.js:453:12)
data:    app.js:1037 -     at Function.Module._load (module.js:445:3)
data:    app.js:1037 -     at Module.require (module.js:504:17)
data:    app.js:1037 -     at require (internal/module.js:20:19)
data:    app.js:1037 -     at Object.<anonymous> (/project/routes/script.js:2:26)
data:    app.js:1037 -     at Module._compile (module.js:577:32)
data:    app.js:1037 -     at Object.Module._extensions..js (module.js:586:10)
data:    app.js:1037 -     at Module.load (module.js:494:32)
data:    app.js:1037 -     at tryModuleLoad (module.js:453:12)
data:    app.js:1037 -     at Function.Module._load (module.js:445:3)
data:    app.js:1037 -     at Module.require (module.js:504:17)
data:    app.js:1037 -     at require (internal/module.js:20:19)
data:    app.js:1037 -     at Object.<anonymous> (/project/app.js:407:27)
data:    app.js:1037 -     at Module._compile (module.js:577:32)
data:    app.js:1037 -     at Object.Module._extensions..js (module.js:586:10)
data:    app.js:1037 -     at Module.load (module.js:494:32)
```

**Comment by hip1310** [comment]
Also got the same error while adding plugin to schema.

/projects/node_modules/mongoose-explain/index.js:36
  if (pluginOptions.aggregate !== false) {
                    ^
TypeError: Cannot read property 'aggregate' of undefined

**Comment by vkarpov15** [comment]
Fixed and 0.2.1 released, thanks for reporting :+1:

As a workaround, pass an an empty object as options: `schema.plugin(explain, {})`
